execute/execute.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import json
+import jsonschema
+import time
+
+from pathlib import Path
+
+from console_access_api.aitrios_console import AitriosConsole
+
+parser = argparse.ArgumentParser(
+    description="Launches execution on a camera configured in AITRIOS Console",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument(
+    "--aitrios_secrets",
+    "-s",
+    type=str,
+    dest="aitrios_secrets",
+    help=f"The path to your AITIRIOS console configuration file",
+    required=True,
+)
+parser.add_argument(
+    "--device_name",
+    "-D",
+    type=str,
+    dest="device_name",
+    help=f"Device Name to search for",
+    required=True,
+)
+
+
+def load_configuration_file(configuration_path: Path, schema_path: Path):
+    """
+    Parse a secrects configuration file, validates against schema
+    """
+    # Load configuration file
+    with open(configuration_path, "r") as f:
+        configuration = json.load(f)
+
+    # Load configuration schema file
+    with open(schema_path, "r") as f:
+        json_schema = json.load(f)
+
+    # Validate configuration
+    jsonschema.validate(configuration, json_schema)
```

Comment:
With `Path` there's an alternate spelling here that is slightly more compact. No strong perference to one over another, though. I would probably leave off the comments

```suggestion
    configuration = json.loads(configuration_path.read_text())
    json_schema = json.loads(schema_path.read_text())

    jsonschema.validate(configuration, json_schema)
```

Reply:
`loads` didn't work for some reason.